### PR TITLE
#163857760 Bootstrap and Fontawesome setup

### PR DESCRIPTION
**Pull request by julietezekwe**
#### What does this PR do?
It sets up the configuration for Bootstrap 4 and font-awesome.

#### Description of Task to be completed?
- Set up bootstrap 4
- Configure webpack fro bootstrap bundling
- configure webpack for font-awesome

#### How should this be manually tested?
In the application, run `npm install`
Apply any bootstrap class and any font-awesome icon

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[163857760](https://www.pivotaltracker.com/story/show/163857760)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
```
Comment (andela-dbamidele) [inline .eslintrc]: Why do you have this file in your PR?

I think you this PR contains some of the files in the previous PR.

**Review by andela-dbamidele — CHANGES_REQUESTED** [review]
Looks good. Please update the PR and remove work from the previous PR. Thanks

File: .eslintrc (hunk 1 above)
Comment (julietezekwe) [reply .eslintrc]: I started working on this branch before the eslint PR was merged and so I had to pull origin staging into this branch and the eslint from staging appeared s a new file.

Comment (julietezekwe) [reply .eslintrc]: I will fix it now.

**Pull request merged**